src/opencensus/proto/trace/v1/trace.proto

```diff
@@ -290,6 +291,11 @@ message Span {
   // Status.Ok (code = 0).
   Status status = 11;
 
+  // An optional resource that this span describes. If not set, this span 
+  // should be part of a batch that does include the span information, unless resource 
```

Comment:
"that does include the span information" or "that does include the **resource** information"?

Reply:
ah you are right, should be resource. -- fixing